activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -432,9 +432,24 @@ def topic.title() "b" end
       assert_equal false, Topic.new(author_name: value).author_name?
     end
 
+    ActiveModel::Type::Boolean::FALSE_VALUES.each do |value|
+      assert_predicate Topic.new(author_name: value), :author_name?
```

Comment:
This assertion is already in master branch. Please rebase this branch on master.